src/main/groovy/com/bmuschko/gradle/docker/utils/DockerThreadContextClassLoader.groovy

```diff
@@ -361,6 +361,23 @@ class DockerThreadContextClassLoader implements ThreadContextClassLoader {
      * {@inheritDoc}
      */
     @Override
+    def createDevice(final String deviceString) {
+        Class deviceClass = loadClass("${MODEL_PACKAGE}.Device")
+        try {
+            // The parse method is  available in newer Docker java library versions.
+            Method method = deviceClass.getMethod("parse", String.class)
```

Comment:
No need to use `String.class`. Simply use `String` and groovy should magically make things happen ;)